WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift

```diff
@@ -42,7 +41,7 @@ struct ShippingLabelPackageList: View {
             .background(Color(.systemBackground))
         }
         .background(Color(.listBackground))
-        .navigationBarTitle(Text(Localization.title), displayMode: .inline)
+        .navigationTitle(Localization.title)
```

Comment:
[Q] Why this change by mistake? 😅

Reply:
Because of this `I also updated the method navigationBarTitle with navigationTitle available on iOS 14.`. It's just a new way to declare the bar title in iOS 14. 👍